sources/osgShadow/ShadowMap.js

```diff
@@ -60,20 +62,29 @@ define( [
             }
             /*jshint bitwise: true */
         },
+        // Visiting whole casting scene
         apply: function ( node ) {
             var st = node.getStateSet();
             // check that and other things ?
+            // TODO: should check whole hierarchy in fact...
             if ( st ) {
+                // check for transparency not casting shadows
+                // as no alpha blending transparency shadow (no transmittance support)
```

Comment:
that would be could to keep this logic into function, if the user wants to override this logic he could